exercises/wordy/wordy_test.rb

```diff
@@ -71,7 +71,8 @@ def test_multiply_twice
   def test_add_then_multiply
     skip
     question = 'What is -3 plus 7 multiplied by -2?'
-    assert_equal(-8, WordProblem.new(question).answer)
+    message = 'You should ignore order of precedence. 3 + 2 * 3 = 15, not 9.'
```

Comment:
It would be nice if the error message matched the question: "-3 + 7 * -2 = -8, not -17"